src/components/views/rooms/RoomSettings.js

```diff
@@ -364,6 +389,11 @@ module.exports = React.createClass({
         var powerLevels = this.props.room.currentState.getStateEvents('m.room.power_levels', '');
         powerLevels = powerLevels ? powerLevels.getContent() : {};
 
+        for (let key of Object.keys(this.refs).filter(k => k.startsWith("event_levels_"))) {
```

Comment:
Given that this would require a lot more work if we used `state`, this is probably fine. @dbkr do you have thoughts on this?

Reply:
tbh I only used refs because the rest of the powerlevels use refs.